readthedocs/core/management/commands/contact_owners.py

```diff
@@ -0,0 +1,167 @@
+
+from readthedocs.core.permissions import AdminPermission
+from readthedocs.core.utils.contact import contact_users
+from readthedocs.organizations.models import Organization
+from readthedocs.projects.models import Project
+
+User = get_user_model()  # noqa
+
+log = logging.getLogger(__name__)
+
+
+class Command(BaseCommand):
+
+    """
+    Send an email or sticky notification from a file (markdown) to all owners.
+
+    Usage examples
+    --------------
+
+    Email all owners of the site::
+
+      django-admin contact_owners --email email.md
+
+    Email and send a sticky notification to all owners of the "readthedocs" organization::
+
+      django-admin contact_owners --email email.md --notification notification.md --organization readthedocs  # noqa
+
+    Where ``email.md`` is a markdown file with the first line as the subject, and the rest is the content.
+
+    .. code:: markdown
+
+       Read the Docs deprecated option, action required
+
+       Dear user...
+
+    .. note::
+
+       By default the command won't send the email/notification (dry-run mode),
+       add the ``--production`` flag to actually send the email/notification.
+    """
+
+    help = 'Send an email or sticky notification from a file (markdown) to all owners.'
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            '--production',
+            action='store_true',
+            dest='production',
+            default=False,
+            help=(
+                'Send the email/notification for real, '
+                'otherwise we only print the notification in the console (dryrun).'
+            )
+        )
+        parser.add_argument(
+            '--email',
+            help=(
+                'Path to a file with the email content in markdown. '
+                'The first line would be the subject.'
+            ),
+        )
+        parser.add_argument(
+            '--notification',
+            help='Path to a file with the notification content in markdown.',
+        )
+        parser.add_argument(
+            '--organization',
+            help='Organization slug to filter by.',
+        )
+        parser.add_argument(
+            '--project',
+            help='Project slug to filter by.',
+        )
+
+    def handle(self, *args, **options):
+        if not options['email'] and not options['notification']:
+            print("--email or --notification is required.")
+            sys.exit(1)
+
+        project = options['project']
+        organization = options['organization']
+        if project and organization:
+            print("--project and --organization can\'t be used together.")
+            sys.exit(1)
+
+        if project:
+            project = Project.objects.get(slug=project)
+            users = AdminPermission.owners(project)
+        elif organization:
+            organization = Organization.objects.get(slug=organization)
+            users = AdminPermission.owners(project)
```

Comment:
Shouldn't this pass `organization` here?

Reply:
sending a fix